_posts/2020-06-23-case-principal-transformer.adoc

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: "Using case principal transformers in Elytron"
+date: 2020-06-23
+tags: wildfly elytron
+synopsis: An overview of the new case principal transformer included in WildFly 21.
+author: szaldana
+link:
+---
+
+The *INSERT_LINK_[case-principal-transformer]*
```

Comment:
Can update this to https://github.com/wildfly-security-incubator/elytron-examples/tree/master/case-principal-transformer (this link will work once we've merged your elytron-examples PR).